src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star + .stars__star {
+  margin-left: 4px;
+}
```

Comment:
`:not` and `:first-child` is more readable and understandable

Reply:
understood, thx